### DISCUSSION TITLE
Any way to cap the Y axis max but keep it auto when its below the cap?

### DISCUSSION THREAD

**Discussion by flybrys** [RomRider/apexcharts-card#714] [Q&A]
I have a sensor that monitors the forecast electricity price for the next 24 hours.
Normal ranges are 0.00 to 0.40 however can go up to 1.00 daily and occasionally a price spike happens where it can go all the way up to 19.00

I use the graph with min and max as auto so I can clearly see when the price will be the cheapest and it works great, however when there is a price spike period forecast, the smaller values show as a flat line because the range is so large.

Is there a way to say, cap the max at 1.20, but still keep it as auto when it is below this value?

Here's a picture of today's forecast, and one where I limit the max to 1.

Max 1:
![image](https://github.com/RomRider/apexcharts-card/assets/54029411/242d008f-14a7-48be-a2f9-12a00f53e14b)

Auto:
![image](https://github.com/RomRider/apexcharts-card/assets/54029411/077e400c-1a00-4f5a-b06c-56edbaa10b86)

**Comment by RomRider** [comment]
Not possible at the moment, but please open a feature request with the same information
